soci/soci_index.go

```diff
@@ -40,23 +40,23 @@ import (
 )
 
 const (
-	// artifactType of index SOCI index
+	// SociIndexArtifactType is the artifactType of index SOCI index
 	SociIndexArtifactType = "application/vnd.amazon.soci.index.v1+json"
-	// mediaType of ztoc
+	// SociLayerMediaType is the mediaType of ztoc
 	SociLayerMediaType = "application/octet-stream"
-	// index annotation for image layer media type
+	// IndexAnnotationImageLayerMediaType is the index annotation for image layer media type
 	IndexAnnotationImageLayerMediaType = "com.amazon.soci.image-layer-mediaType"
-	// index annotation for image layer digest
+	// IndexAnnotationImageLayerDigest is the index annotation for image layer digest
 	IndexAnnotationImageLayerDigest = "com.amazon.soci.image-layer-digest"
-	// index annotation for build tool identifier
+	// IndexAnnotationBuildToolIdentifier is the index annotation for build tool identifier
 	IndexAnnotationBuildToolIdentifier = "com.amazon.soci.build-tool-identifier"
-	// media type for OCI Artifact manifest
+	// OCIArtifactManifestMediaType is the media type for OCI Artifact manifest
 	OCIArtifactManifestMediaType = "application/vnd.oci.artifact.manifest.v1+json"
-	// default span size (4MiB)
+	// defaultSpanSize (4MiB)
```

Comment:
This documentation doesn't look clearer than the before.

Reply:
This more follows [go doc comment](https://go.dev/doc/comment) that comments start with name. I'm okay to change to back since it's not exported.